packages/@aws-cdk/aws-ecr/lib/repository.ts

```diff
@@ -490,6 +514,7 @@ export class Repository extends RepositoryBase {
         scanOnPush: true,
       },
       imageTagMutability: props.imageTagMutability || undefined,
+      encryptionConfiguration: encryptionConfiguration,
```

Comment:
There is no point to create the const in line 506 since it is only used here:

```suggestion
      encryptionConfiguration: this.parseEncryption(props),
```